fling/component/client/fedamp_client.py

```diff
@@ -0,0 +1,54 @@
+import copy
+import numpy as np
+from typing import Tuple
+
+import torch
+import torch.nn as nn
+
+from fling.utils.registry_utils import CLIENT_REGISTRY
+from .base_client import BaseClient
+from fling.utils.utils import weight_flatten
+
+@CLIENT_REGISTRY.register('fedamp_client')
+class FedAMPClient(BaseClient):
```

Comment:
I'm wondering whether this client is identical to ``FedProxClient``? What's the differences?